src/main/java/io/jenkins/plugins/artifact_manager_jclouds/BlobStoreProvider.java

```diff
@@ -51,21 +51,21 @@
         GET, PUT;
     }
 
-    /** A constant for the blob path prefix to use. */
+    /** A constant for the artifact path prefix to use. */
     @NonNull
     public abstract String getPrefix();
 
-    /** A constant for the blob container name to use. */
+    /** A constant for the artifact container name to use. */
```

Comment:
Well, here “blob” actually made sense. The blob container can store various things, currently artifacts and stashes.